MathDisplay/MathAtom/LaTeX.fs

```diff
@@ -75,6 +80,83 @@ let toAtom (settings: Options) latex =
         ///Processes 2 arguments, then continues reading
         let arg2 = argPlus1 arg1
 
+        let processCommand cmd cs =
+            match cmd with
+            | _ ->
```

Comment:
looks a bit odd

Reply:
Originally intended for '^', '_', etc. Now deleted.